### DISCUSSION TITLE
module 'lua-dev' not found

### DISCUSSION THREAD

**Discussion by pijunhuis** [nshen/learn-neovim-lua#137] [Q&A] [answered]
<img width="972" alt="image" src="https://user-images.githubusercontent.com/24618917/200106711-55f682f9-d785-4fe7-adef-37274dbd2cae.png">

**Comment by pijunhuis** [comment, marked as answer]
已解决

**Reply by Fisherder** [reply]
我也遇到了这个问题，请问您是怎么解决的？
